src/sections/Route.enum.ts

```diff
@@ -3,5 +3,6 @@ export enum Route {
   SIGN_UP = '/dataverseuser.xhtml?editMode=CREATE&redirectPage=%2Fdataverse.xhtml',
   LOG_IN = '/loginpage.xhtml?redirectPage=%2Fdataverse.xhtml',
   LOG_OUT = '/',
-  DATASETS = 'datasets'
+  DATASETS = 'datasets',
+  CREATE = 'create'
```

Comment:
What about 'createDataset' Collections will also have to be created, not just datasets.

Reply:
@MellyGray  suggested we use the url path `datasets/create` and likely `dataverse/create`. Otherwise my first thought was similar, to have a `createDataset` and `createDataverse`. Although I am unsure whether we are differentiating between datasets and dataverse routes at this time?